config/checkstyle/checkstyle-suppressions.xml

```diff
@@ -11,6 +11,9 @@
     <suppress files="[/\\]src[/\\].*[Tt]est[/\\](java|groovy)[/\\]" checks="VariableDeclarationUsageDistance" />
     <suppress files="[/\\]src[/\\].*[Tt]est[/\\](java|groovy)[/\\]" checks="VisibilityModifier" />
     <suppress files="[/\\]src[/\\].*[Tt]est[/\\](java|groovy)[/\\]" checks="AvoidStaticImport" />
+    <suppress files="[/\\]src[/\\].*[Tt]est[/\\](java|groovy)[/\\]" checks="AbbreviationAsWordInName" />
+    <suppress files="[/\\]src[/\\].*[Tt]est[/\\](java|groovy)[/\\]" checks="MemberName" />
+    <suppress files="[/\\]src[/\\].*[Tt]est[/\\](java|groovy)[/\\]" checks="MethodName" />
```

Comment:
Again, I'd prefer to isolate these to single spots if possible. See comment [here](https://github.com/hashgraph/hedera-services/pull/14685#discussion_r1709811671) for reference

Reply:
Same as [here](https://github.com/hashgraph/hedera-services/pull/14685#discussion_r1711101452)